.github/workflows/qa-testbench.yaml

```diff
@@ -102,23 +112,33 @@ jobs:
     name: Send slack notification on build failure
```

Comment:
:upside_down_face: The name and description isn't accurate anymore, we always send a slack notification.